### PR TITLE
Add Answers to FAQ

### PR DESCRIPTION
**Pull request by danintel**
Signed-off-by: danintel <daniel.anderson@intel.com>

### CODE REVIEW

File: docs/FAQ.rst [hunk 1]
```diff
@@ -107,13 +107,58 @@ https://github.com/hyperledger/avalon/issues
...
+workorder, so the orchestrator app can explicitly remove completed workorders.
+This is especially useful for to remove large, obsolete responses so as to not
+exhaust the database. This API mitigates the LMDB size but is not intended as
+a replacement for auto-clean up.
+
+In the future, a clean up tool may be added.
+
+Does Avalon support a Library OS (LibOS)?
+-----------------------------------------
+Avalon does not support a LibOS now, but support is planned in the future,
```
Comment (rranjan3) [inline docs/FAQ.rst:144]: Can the advantage of using a LibOS also be highlighted here?

File: docs/FAQ.rst [hunk 2]
```diff
@@ -107,13 +107,58 @@ https://github.com/hyperledger/avalon/issues
...
+The only potential external use that should be made of LMDB is by an
+Avalon hosting service for its monitoring and administration tools.
+
+Is there a mechanism to auto-trim the LMDB database?
+----------------------------------------------------
+Avalon is supposed to auto-trim LMDB after exceeding a maximum threshold
+by removing old entries. It may not be fuly implemented yet,
+but that is the plan. There is also an explicit API to delete a specific
+workorder, so the orchestrator app can explicitly remove completed workorders.
+This is especially useful for to remove large, obsolete responses so as to not
```
Comment (rranjan3) [inline docs/FAQ.rst:136]: This is especially useful _for to remove_ -> This is especially useful _for removing_

File: docs/FAQ.rst (hunk 1 above)
Comment (danintel) [reply docs/FAQ.rst:144]: @rranjan3 I added LibOS advantages to the glossary, below.

File: docs/FAQ.rst (hunk 2 above)
Comment (danintel) [reply docs/FAQ.rst:136]: @rranjan3 fixed

File: docs/FAQ.rst [hunk 3]
```diff
@@ -107,13 +107,58 @@ https://github.com/hyperledger/avalon/issues
...
 No other Avalon component connects to the blockchain.
 
 Another Avalon instance unrelated to any other Avalon instance may connect
 independently to the blockchain.
 
+What synchronization modes does Avalon Support?
+-----------------------------------------------
+The Off-Chain Trusted Compute Spec has three modes (asynchronous,
+notify, and synchronous). Avalon currently supports async mode.
+Synch mode may be implemented, depending on interest.
```
[the remark this answers is not in the document]
Comment (danintel) [reply docs/FAQ.rst:121]: @rranjan3 Added Singleton mode with ZMQ

**Pull request merged**